packages/cozy-stack-client/src/PermissionCollection.js

```diff
@@ -155,6 +171,18 @@ class PermissionCollection extends DocumentCollection {
   }
 }
 
+/**
+ * Is the document a collection document?
+ * @param {{_type}} document
+ * @returns {boolean}
+ */
+export const isCollection = document => {
+  if (isFile(document)) return false
+  if (document._type == 'io.cozy.notes') return false
+  if (document._type == 'io.cozy.notes.events') return false
```

Comment:
should we create an `isNote(document)` testing if _type starts with `io.cozy.notes` ? 

Also it's a bit disturbing that sometime a note is a `io.cozy.file` and sometimes (in the realtime use case) we work with an `io.cozy.notes`

Reply:
We always work with a io.cozy.files.
The couchdb io.cozy.notes exists but we shouldn't have to use it.

The addition here is more because I was on the function and it was free to fill some examples.